sdk/servicebus/service-bus/src/index.ts

```diff
@@ -43,7 +43,6 @@ export {
 export { OperationOptions } from "./modelsToBeSharedWithEventHubs";
 
 export { Receiver } from "./receivers/receiver";
-export { SubscriptionRuleManager } from "./receivers/subscriptionRuleManager";
```

Comment:
You need to remove `CorrelationFilter` and other rule related interfaces from the api as well

Reply:
Yes, updated. Anything else?

Reply:
The .md file didnt get updated :)

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-js/pull/8660/commits/c057589420d94ab7daae6e6538ffcae488e3c986

Reply:
Ah thanks.